code/game/verbs/ooc.dm

```diff
@@ -255,3 +255,30 @@ var/global/normal_ooc_colour = "#002eb8"
 
 		pct += delta
 		winset(src, "mainwindow.split", "splitter=[pct]")
+
+
+/client/verb/update_ping(time as num)
```

Comment:
```suggestion
/client/verb/update_ping(time)
```

Is num required here because its a verb?

Reply:
Not required, good for clarity.